README.md

```diff
@@ -228,3 +230,20 @@ $spacing: (
 ```
 
 The exact names of the keys in this map aren't important, as long as `@include spacing(foo)` has a matching key `foo` in the map.
+
+### Fluid Type
+Written by [Indrek Paas](https://github.com/indrekpaas) and inspired by [Mike Riethmuller](https://madebymike.com.au/writing/precise-control-responsive-typography/) this mixin allows you to specify a minumum and maximum size for values. Allowing for more fluid websites.
+
+#### As an SCSS mixin
+```
```

Comment:
One small thing I unfortunately didn't notice until after merging, @brianjhanson 

These three ticks should have `scss` at the end so that syntax highlighting will work correctly.